virttest/iscsi.py

```diff
@@ -833,5 +834,170 @@ def create_iSCSI(params, root_dir=data_dir.get_tmp_dir()):
                 )
             iscsi_instance = IscsiTGT(params, root_dir)
         else:
-            iscsi_instance = IscsiLIO(params, root_dir)
+            iscsi_instance = (
+                MultiTargetsIscsiLIO(params, root_dir)
+                if not params.get("target") and params.get("targets_luns")
+                else IscsiLIO(params, root_dir)
+            )
         return iscsi_instance
+
+
+class MultiTargetsIscsiLIO(IscsiLIO):
+    """
+    Iscsi class supporting multi-targets for the LIO backend used in RHEL7.
+    """
+
+    def __init__(self, params, root_dir):
+        """
+        Init the object based on the params and root_dir.
+        Note: 'targets_luns' in params is introduced at this class.
+              The structure of 'targets_luns':
+                targets_luns =
```

Comment:
Would you give an example? i.e. how to set iscsi_targets_luns in case.

One more thing, if we add a new cartesian param, it's better to add it in some cfg file, e.g. under iscsi backend in the file guest-hw.cfg , the problem is we don't support it for the iscsi storage backend currently, it might be OK to comment it here

Reply:
> Would you give an example? i.e. how to set iscsi_targets_luns in case.
> 
> One more thing, if we add a new cartesian param, it's better to add it in some cfg file, e.g. under iscsi backend in the file guest-hw.cfg , the problem is we don't support it for the iscsi storage backend currently, it might be OK to comment it here

Hello @zhencliu thanks to the comment. I move those comments to cfg.file